modules/ROOT/pages/auth.adoc

```diff
@@ -0,0 +1,170 @@
+= Luna Streaming Authentication
+
+== Authentication
+
+The Helm chart can enable token-based authentication for your Pulsar cluster. For information on token-based
+authentication in Pulsar, go https://pulsar.apache.org/docs/en/security-token-admin/[here].
+
+For authentication to work, the token-generation keys need to be stored in Kubernetes secrets along with some default tokens (for superuser access). 
```

Comment:
What does superuser apply to? Is it required for any authentication? Or are the default tokens used for superuser access?